app/buyers/views/buyers.py

```diff
@@ -127,6 +127,10 @@ def view_brief_overview(framework_slug, lot_slug, brief_id):
     sections = content.summary(brief)
     delete_requested = True if request.args.get('delete_requested') else False
 
+    content_loader.load_messages(brief['frameworkSlug'], ['urls'])
+    call_off_contract_url = content_loader.get_message(brief['frameworkSlug'], 'urls', 'call_off_contract_url')
```

Comment:
See if you can find another place where we rely on urls (maybe signing framework agreements) and follow their convention. I'm happy either way as we aren't being less protective then we are in other places